corehq/apps/analytics/ab_tests.py

```diff
@@ -1,53 +1,128 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
-import collections
 import random
+from django.core.cache import cache
 
 from memoized import memoized
 from corehq.apps.accounting.models import Subscription
 
-ABTestConfig = collections.namedtuple('ABTestConfig', ('name', 'slug', 'options'))
 
+class SessionAbTestConfig(object):
```

Comment:
Thank you for adding all the docs in this file. I think it'd be helpful to make a note in https://github.com/dimagi/commcare-hq/tree/master/corehq/apps/analytics#handling-different-environments-and-debugging that there are docs in this file relevant to debugging.